apps/jubmoji-quest/src/pages/collect/index.tsx

```diff
@@ -106,7 +106,7 @@ export default function CollectJubmojiPage() {
   const params = useParams();
   const [isModalOpen, setIsModalOpen] = useState(true);
 
-  const { data: jubmojiCollectionCards = [], isLoading } = useCards();
+  const { data: jubmojiCollectionCards = [], isLoading } = useFetchCards();
```

Comment:
Initial collection view emoji is off center?

![IMG_6840](https://github.com/jubmoji/jubmoji.quest/assets/6290822/cd7a22a2-4723-4125-b2cd-04603bfe881c)

Reply:
fixed!